src/Microsoft.Diagnostics.Runtime/src/Extensions/IAddressableTypedEntityExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Diagnostics.Runtime.Extensions
```

Comment:
I don't think we should add a new namespace here.  No one will find this extension method.  Put this in `Microsoft.Diagnostics.Runtime` instead.

Reply:
With that said, I do like putting in "src\Extensions", that does seem like the right place for it, even if not using a new namespace.

Reply:
Yep, I had similar idea. Thanks.